tmt/steps/provision/minute.py

```diff
@@ -397,6 +397,7 @@ def start(self):
         if self.opt('dry'):
             return
         self.mt_image = self._convert_image(self.image)
+        self.debug(f"Using image: {self.mt_image}")
```

Comment:
Would it make sense to include the final image in the verbose (or even info) output? Could be useful especially when called with `--image fedora` without distro version.

Reply:
Hm, isn't this usecase covered by what provision step outputs? If I run `tmt run provision -h minut finish`, one of the output lines is `        distro: Fedora 33 (Thirty Three)`. Verbose sounds good to me, looking at other provision plugins, something like `self.verbose('1mt_image', self.mt_image, 'green')`?

Reply:
Or make it info/verbose if `_convert_image` changed the image name?

Reply:
I went with @FrNecas' proposal.